### ISSUE TITLE
📝 Docs: Add documentation to Google calendar

### ISSUE THREAD

**Issue by harshithmullapudi** [poozlehq/poozle#72]
Add documentation for Google calendar to show up at https://docs.poozle.dev/extensions/google-calendar similar to https://docs.poozle.dev/extensions/github

**Comment by OzairKhanBro** [comment]
Can you please assign this ticket to me as well?
The provided link is not correct. I guess you are referring https://docs.poozle.dev/ticketing/integrations/github

**Comment by saimanoj** [comment]
@OzairKhanBro Yeah, you've referred to it correctly. I'm assigning it to you. if you've any questions, feel free to ping us on Slack.

**Comment by OzairKhanBro** [comment]
Thanks.
